src/libais/ais.h

```diff
@@ -2433,6 +2433,23 @@ class Ais27 : public AisMsg {
 };
 ostream& operator<< (ostream &o, const Ais27 &msg);
 
+//
+// SSW Satellite Ship Weather 1-Slot Version
+class Ais8_367_24 : public Ais8 {
+ public:
+  int version;
```

Comment:
I'm switching to default member initializer style, so please change to `int version = 0;` and similar for all the data members.